app/forms/steps/capital/other_property_type_form.rb

```diff
@@ -0,0 +1,11 @@
+module Steps
+  module Capital
+    class OtherPropertyTypeForm < PropertyTypeForm
+      private
+
+      def persist!
+        @property = crime_application.properties.create(property_type:)
```

Comment:
We've got `create!` for savings and investments loops now if you want to keep it consistent

Reply:
Could you update `property_type_form` too please?